script.js

```diff
@@ -0,0 +1,74 @@
+// Global Variables
+
+var counter = 1;
+
+// Adds 'x' or 'o' to the HTML of the corresponding button
+// btnNum is a variable set to the Event Listener
+function markMove(btnNum){
+	counter += 1;
+	var button = document.getElementById(btnNum);
```

Comment:
Good job breaking this out into a function (rather than repeating it in each event listener). To satisfy the requirement that you can't play the same square twice, could you add an 'if statement' here to check whether the square already has an x or o inside of it?